app-utils/hw-probe/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=hw-probe
+PKGSEC=utils
+PKGDEP="perl hwinfo curl dmidecode smartmontools pciutils usbutils edid-decode"
+PKGDES="Probe for hardware, check operability and find drivers"
```

Comment:
A tool to probe, test, and find drivers for your hardware

Reply:
done